app/views/posts/index.html.erb

```diff
@@ -1 +1,50 @@
-<h1>Display All Post</h1>
+<section>
+  <h1>List of Posts for spesfic User</h1>
+
+<div class="d-flex align-items-center mb-3 gap-3 text-secondary-emphasis w-100">
+  <div>
+    <img src="<%= @user.photo %>" alt="user photo" class= "border border-dark border-3 p-5">
+  </div>
+  <div class=" border border-dark border-3 w-100 p-3">
```

Comment:
- [ ] Again here great work implementing the `see all post` button, but kindly style the post index view to match the given design

Given design
![image](https://user-images.githubusercontent.com/81932478/230741771-b9ff448e-865f-40aa-99aa-11c3ca18aa4a.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/230741815-fdf0b27f-2895-498d-a20a-32f80d8a509e.png)